tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/globalvar/GlobalVarNegativeTest.java

```diff
@@ -134,8 +134,9 @@ public void testConfigurableModuleVarDeclNegative() {
                 "union member type '(json & readonly)' is not supported", 93, 1);
```

Comment:
Shall we create an issue for support json types

Reply:
created https://github.com/ballerina-platform/ballerina-lang/issues/35536